provisioning/openstack/CREATE.md

```diff
@@ -14,49 +14,21 @@ To use the PNDA command line interface, you will need to install the python, hea
 
 ```
 sudo yum install -y epel-release
-sudo yum install -y python python-pip python-devel
+sudo yum install -y python python-pip python-devel gcc
 
 cd cli
 sudo pip install -r requirements.txt
 sudo pip install jinja2 --upgrade
 ```
-#### Keystone authentication
-
-You must authenticate with Keystone before using the OpenStack clients.
-
-The easiest way to accomplish this is to download the credentials file usually named <project>-openrc.sh from Horizon for the given user and tenant, then source it in the shell you will use to create PNDA.
-
-```
-. <project>-openrc.sh
-```
 
 #### CLI invocation
 
-The heat_cli.py scripts allows to launch a PNDA deployment. It sits in the cli subdirectory.
-
-
```

Comment:
You've completely removed an example invocation of the CLI, which is the primary purpose of this page, and deleted the statement about -s which is a major source of error.

Note also that the pnda-cli repo itself currently states - "At present, the PNDA CLI supports two back ends"